### PR TITLE
backend-test: extend create_org test with additional checks

### PR DESCRIPTION
**Pull request by kjaskiewiczz**
changes:
- check the device limit for default plan ("os")
- create "professional" tenant and check the limit
- try to create org with invalid plan and check the error

Issue: https://tracker.mender.io/browse/MEN-3246

### CODE REVIEW

**Comment by kjaskiewiczz** [comment]
related PRs: https://github.com/mendersoftware/mender-conductor-enterprise/pull/41, https://github.com/mendersoftware/tenantadm/pull/126

**Comment by kjaskiewiczz** [comment]
and backend integration tests run for all those PRs: https://gitlab.com/Northern.tech/Mender/mender-qa/pipelines/121632171

File: backend-tests/tests/test_create_org.py [hunk 1]
```diff
@@ -85,6 +88,75 @@ def test_success(self, clean_migrated_mongo):
...
+        assert r.status_code == 200
+        assert r.json()["limit"] == 50
+
+    def test_success_with_plan(self, clean_migrated_mongo):
+        tc = ApiClient(tenantadm.URL_MGMT)
+        uc = ApiClient(useradm.URL_MGMT)
+        tenantadmi = ApiClient(tenantadm.URL_INTERNAL)
+        devauthi = ApiClient(deviceauth_v1.URL_INTERNAL)
+
+        logging.info("Starting TestCreateOrganizationEnterprise")
```
Comment (lluiscampos) [inline backend-tests/tests/test_create_org.py]: This logging message is a bit off.

File: backend-tests/tests/test_create_org.py [hunk 2]
```diff
@@ -85,6 +88,75 @@ def test_success(self, clean_migrated_mongo):
...
+                auth=("some.user@example.com", "asdfqwer1234"),
+            )
+            if rsp.status_code == 200:
+                break
+            time.sleep(1)
+
+        if rsp.status_code != 200:
+            raise ValueError(
+                "User could not log in within five minutes after organization has been created."
+            )
```
Comment (lluiscampos) [inline backend-tests/tests/test_create_org.py]: This is duplicated in the previous test case, with the only exception of here waiting 5 minutes and there 3. Can we refactor it? Even better if we could move it to a common place for other tests to possibly use.

**Pull request merged**